src/visitors/defuse_analyze_visitor.hpp

```diff
@@ -50,6 +50,14 @@ enum class DUState {
     NONE
 };
 
+/**
+ * Variable type processed by DefUseAnalyzeVisitor
+ *
+ * DUVariableType::Local means that we are looking for LD, LU and CD DUStates, while Global means we
+ * are looking for U, D and CD DUStates.
```

Comment:
So `and CD DUStates` is common for local as global variables, right?

Reply:
Yes, `CD` is same for `Local` and `Global` vars as well. I will add this in the documentation of the state as well :+1: